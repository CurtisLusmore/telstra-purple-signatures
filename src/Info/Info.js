import React from 'react';

const Info = () => (
  <div>
    <h3>Create your signature</h3>
    <ol>
      <li>Enter your details.</li>
      <li>Click the "Copy Signature" button</li>
      <li>Open outlook</li>
      <li>
        <a
          style={{ fontWeight: 'bold', textDecoration: 'underline' }}
          href="https://support.office.com/en-us/article/create-and-add-a-signature-to-messages-8ee5d4f4-68fd-464a-a1c1-0e1c80bb27f2"
          rel='noopener noreferrer'
          target="_blank">
          Paste selection into signature editor
        </a>
        <ul style={{ paddingLeft: '1rem', listStyleType: 'none', paddingTop: '0.5rem' }}>
          <li style={{ marginBottom: '0.5rem' }}>
            <span style={{ fontWeight: 'bold' }}>Windows: </span>
            Signature > Signatures from the Message menu
          </li>
          <li>
            <span style={{ fontWeight: 'bold' }}>Mac: </span>
            Preferences > Signatures from the Message menu
          </li>
        </ul>
      </li>
    </ol>
    <br/>
  </div>
);

export default Info;