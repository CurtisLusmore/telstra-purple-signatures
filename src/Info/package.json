{
  "main": "./Info.js",
  "name": "Info",
  "version": "0.0.0",
  "private": true
}
