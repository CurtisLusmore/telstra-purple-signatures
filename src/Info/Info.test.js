import React from 'react';
import { shallow } from 'enzyme';

import Info from './Info';

it('Renders', () => {
  const tree = shallow(<Info/>);
  expect(tree).toMatchSnapshot();
});