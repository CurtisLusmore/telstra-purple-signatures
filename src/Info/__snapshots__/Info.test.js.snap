// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div
  className="content"
>
  <h3
    className="title"
  >
    Create your signature
  </h3>
  <ol>
    <li>
      Enter your details.
    </li>
    <li>
      Click the "Copy Signature" button
    </li>
    <li>
      Open outlook
    </li>
    <li>
      <a
        href="https://support.office.com/en-us/article/create-and-add-a-signature-to-messages-8ee5d4f4-68fd-464a-a1c1-0e1c80bb27f2"
        rel="noopener noreferrer"
        style={
          Object {
            "fontWeight": "bold",
            "textDecoration": "underline",
          }
        }
        target="_blank"
      >
        Paste selection into signature editor
      </a>
      <ul
        style={
          Object {
            "listStyleType": "none",
            "margin": "0",
            "paddingLeft": "1rem",
            "paddingTop": "0.5rem",
          }
        }
      >
        <li
          style={
            Object {
              "marginBottom": "0.5rem",
            }
          }
        >
          <span
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
            Windows: 
          </span>
          File &gt; Options &gt; Mail &gt; Signatures
        </li>
        <li>
          <span
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
            Mac: 
          </span>
          Outlook &gt; Preferences &gt; Signatures
        </li>
        <li>
          <span
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
            Outlook web access (OWA):
             
          </span>
          Cog (top-right) &gt; &quot;View all Outlook settings&quot; link (bottom) &gt; Mail &gt; Compose and reply &gt; Email signature
        </li>
      </ul>
    </li>
  </ol>
  <br />
</div>
`;
